layouts/shortcodes/python-venv.html

```diff
@@ -0,0 +1,24 @@
+<p>
+    For Python, before we deploy the stack, the following commands need to be
+    run to create a virtual environment, activate it, and install dependencies:
+</p>
+
+<p>Create a virtual environment:</p>
+<div class="highlight">
+    <pre class="chroma"><code class="language-bash" data-lang="bash">$ python3 -m venv venv</code></pre>
+</div>
+
+<p>Activate the environment in Linux and macOS:</p>
+<div class="highlight">
+    <pre class="chroma"><code class="language-bash" data-lang="bash">$ source venv/bin/activate</code></pre>
+</div>
+
+<p>Activate the environment in Windows:</p>
+<div class="highlight">
+    <pre class="chroma"><code class="language-bash" data-lang="bash">$ venv\Scripts\activate</code></pre>
```

Comment:
> * The reason for `bash` is because on those pages, for whatever reason, `language-batch` is not class we can use.

The reason `language-batch` (or `language-bat`) isn't displayed here is due to a bug in how we hide/show languages when there is a langchooser on the page. We need to fix the `chooser.js` script to ignore more "shell languages" when showing/hiding languages (right now it's only ignoring `bash` and `sh`):

https://github.com/pulumi/docs/blob/4101d892b121505f1900b401a1a1e5d0799a53e0/assets/js/chooser.js#L59

Probably easiest to use the same list of languages we use for the copy button and noselect:

https://github.com/pulumi/docs/blob/4101d892b121505f1900b401a1a1e5d0799a53e0/assets/js/copybutton.js#L27-L55

https://github.com/pulumi/docs/blob/4101d892b121505f1900b401a1a1e5d0799a53e0/assets/js/noselect.js#L15-L16

Once that's done, this could be changed to the following:

```suggestion
    <pre class="chroma"><code class="language-bat" data-lang="bat"><span class="p">&gt;</span> venv\Scripts\activate</code></pre>
```

Reply:
Thanks for letting me know where that is!